Tests/.swiftlint.yml

```diff
@@ -0,0 +1,12 @@
+disabled_rules:
+  - line_length
+  - type_name
+  - valid_docs
+  - force_unwrapping
+  - force_cast
+  - force_try
+  - function_body_length
```

Comment:
I've decided to add this rule because it's tedious to comment every `spec()` func in the tests.